src/LiveComponent/src/EventListener/LiveComponentSubscriber.php

```diff
@@ -313,7 +313,7 @@ private function createResponse(MountedComponent $mounted): Response
 
     private function isLiveComponentRequest(Request $request): bool
     {
-        return 'ux_live_component' === $request->attributes->get('_route');
+        return str_starts_with($request->attributes->get('_route'), 'ux_live_component');
```

Comment:
This assumes a user will start their custom route name with `ux_live_component`?

Reply:
Yes or we can loop thru all Live components and find if a route match. WDYT ?

Reply:
Hmm I think we can do that when compiling the container to create a list of live component routes. Then we just see if the current route is in this compiled list.

@weaverryan, any other ideas?

Reply:
Could it be as simple as having the JS send a custom header and not caring about the route name?

Reply:
Both solutions are great. @weaverryan any ideas ?

Reply:
Hmmm. As the AJAX endpoints return HTML, it should be legal to make a request to the URL with the normal `Content-Type: text/html` header / just put the URL in your browser and have it work. So I think we need to do the normal complex container compilation step :). 

Or, 3rd option. The current route looks like this:

```
$routes->add('ux_live_component', '/{component}/{action}')
```

We could change `{component}` to `{_live_component}` and `{action}` to `{_live_action}`. Then we would look for  a`_live_component` attribute to determine if this is a "live component request". 

Also, thinking about the workflow, I guess you would create your custom route... in YAML somewhere with no controller? That's fine - it feels a bit odd, but I can't think of a better way.

Reply:
I just figure another problem with Ryan's option: we can't rename `{component}` to `{_live_component}` and `{action}` to `{_live_action}`, because `{component}` and `{action}` are also related to TwigComponent ...

@kbond would you like to continue this PR ?

Reply:
What do you mean by this?

> `{component}` and `{action}` are also related to TwigComponent

Reply:
Nevermind, I forgot to clear my project cache after my changes.
PR updated with Ryan's proposal 3.